contracts/oracle/IVoltSystemOracle.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+/// @notice interface for the Volt System Oracle
+interface IVoltSystemOracle {
+    // ----------- Getters -----------
+
+    /// @notice function to get the current oracle price for the entire system
+    function getCurrentOraclePrice() external view returns (uint256);
+
+    /// @notice start time at which point interest will start accruing, and the
+    /// current ScalingPriceOracle price will be snapshotted and saved
+    function periodStartTime() external view returns (uint256);
```

Comment:
add a note here or in docs that if you are past period start time + timeframe, interest doesn't accrue to the current price until you call compound interest